### PR TITLE
Ex6 getPostWithName

### PR DESCRIPTION
**Pull request by mgilyeat**
Search a json file using a post id to list the post and the user to whom it belongs.

### CODE REVIEW

File: students/8/main.js [hunk 1]
```diff
@@ -31,13 +31,33 @@ const ex5 = async () => {
...
 
+// Ex 6
+
+const getPostWithName = async (Id) => {
+    let postName = null;
+    try {
+        let post = await axios.get(`https://jsonplaceholder.typicode.com/posts/${Id}`);
+        retval = post.data
+        user = post.data.userId
+        let userdata = await axios.get(`https://jsonplaceholder.typicode.com/users/${user}`);
```
Comment (mburolla) [inline students/8/main.js]: I think we can delete lines 48,49 and change line 50 to: 

` let userdata = await axios.get(`https://jsonplaceholder.typicode.com/users/${post.data.userId}`);`

and change 52 to:

`postName = Object.assign(post.data, retval2)`

File: students/8/main.js [hunk 2]
```diff
@@ -31,13 +31,30 @@ const ex5 = async () => {
...
 //
 
+// Ex 6
+
+const getPostWithName = async (Id) => {
+    let postName = null;
+    try {
+        let post = await axios.get(`https://jsonplaceholder.typicode.com/posts/${Id}`);
+        let userdata = await axios.get(`https://jsonplaceholder.typicode.com/users/${post.data.userId}`);
+        retval2 = { name: userdata.data.name}
```
Comment (mburolla) [inline students/8/main.js]: retval2 looks kinda dorky here... I would probably get rid of it and do something like this:

`postName = Object.assign(post.data, { name: userdata.data.name})`

Comment (mgilyeat) [reply students/8/main.js]: yeah it does

**Comment by mburolla** [comment]
Undorkified!  👍

**Pull request merged**